spec/build-spec.js

```diff
@@ -35,6 +37,8 @@ describe('Build', () => {
         return specHelpers.vouch(fs.realpath, dir);
       }).then( (dir) => {
         directory = dir + '/';
+        const createdHomeDir = temp.mkdirSync('atom-build-spec-home');
+        os.homedir = () => createdHomeDir;
```

Comment:
I think this should be made async just as it is done a few lines up, also - run it through `fs.realpath` if the temporary directory is a symlink